src/Kunstmaan/NodeBundle/Resources/views/NodeAdmin/Modals/_pagetemplate.html.twig

```diff
@@ -0,0 +1,34 @@
+{% import _self as macros %}
+
+{% for row in pageTemplate.rows %}
+<div class="row">
```

Comment:
Hmmmm, and what is the original problem? Is it that the generated source is not aligned nicely?